lms/djangoapps/courseware/config.py

```diff
@@ -0,0 +1,19 @@
+"""
+Waffle flags and switches
+"""
+
+from edx_toggles.toggles import WaffleSwitch
+
+# Namespace
+WAFFLE_NAMESPACE = 'courseware'
```

Comment:
why is this added to courseware? The financial aid form does not appear in courseware experince.

Reply:
Financial Assistance tools/button is actually part of the course tools which is the part of courseware. I think it makes the most sense to keep it here.

Reply:
No, the button appears on the course outline page, which is technically not a part of the courseware experience.

Reply:
As discussed internally, we're keeping all the configurations for FA in courseware for now.

Reply:
Please record the decision on the ticket alongside the proper slack channel conversation transcripts.